src/Symfony/Component/Cache/DependencyInjection/CachePoolPass.php

```diff
@@ -52,8 +52,8 @@ public function process(ContainerBuilder $container)
             $seed = '.'.$container->getParameterBag()->resolveValue($container->getParameter('cache.prefix.seed'));
         } else {
             $seed = '_'.$container->getParameter('kernel.project_dir');
+            $seed .= '.'.$container->getParameter('kernel.container_class');
```

Comment:
I also think we should set this combined seed as default in https://github.com/symfony/symfony/blob/f46ab58bcff6305331678892c3b4f9f3cf34a15d/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L872
Then the default behavior is obvious and people understand why they might want to change this value.